app/src/main/java/at/roteskreuz/stopcorona/model/receivers/ExposureNotificationBroadcastReceiver.kt

```diff
@@ -33,9 +33,32 @@ import timber.log.Timber
 /**
  * Broadcast receiver for callbacks from exposure notification API.
  */
-class ExposureNotificationBroadcastReceiver : BroadcastReceiver(), KoinComponent {
+class ExposureNotificationBroadcastReceiver : BroadcastReceiver() {
 
-    private val workManager: WorkManager by inject()
+    companion object : KoinComponent {
```

Comment:
Static injecting brokes whole DI flow. If you need workManager inside, provide it as a parameter and inject it in the caller location.

Reply:
ok